src/components/Footer/SoilFooterWithContacts.tsx

```diff
@@ -5,10 +5,16 @@ import plushLogo from '@/assets/images/footer-logo.png'
 import { Link } from 'react-router-dom'
 
 export const SoilFooterWithContacts: FC = () => {
+
+  const PLUSH_LINK = window.config.PLUSH_LINK ?? import.meta.env.VITE_PLUSH_LINK
+
+  const footerLinkHandler = () => {
+    window.open(PLUSH_LINK, '_blank');
+  }
   return (
     <div className={s.soilFooterWithContactsContainer}>
       <div className={s.soilFooterContent}>
-        <div className={s.leftPull}>
+        <div onClick={() => footerLinkHandler()} className={s.leftPull}>
```

Comment:
If no arguments passed, then you can just use onClick={handleFotterLink}. Use verbs in naming of methods, functions.